### ISSUE TITLE
Update contact information

### ISSUE THREAD

**Issue by namurphy** [heliophysicsPy/heliophysicsPy.github.io#6]
I just noticed that the content on [http://heliopython.org/contact/](http://heliopython.org/contact/) is:
```
Contact content goes here. My e-mail is [email@something.com](mailto:email@something.com). 
```
It would be helpful to add an email address or contact person for this effort.

Thanks!
-Nick

**Comment by ehsteve** [comment]
@alex-aquila would you be willing to add your email here and mention that you should be emailed to join the mailing list? I'd like to also add a link to the riot channel here.

**Comment by alex-aquila** [comment]
I think I just did this - let me know if it worked or not!

**Comment by ehsteve** [comment]
Closed with PR #11.
